packages/actions/log/src/components/Log.js

```diff
@@ -0,0 +1,93 @@
+import {FormattedMessage} from 'react-intl'
+import {EditableValue, Button, LoadMask, StatedValue} from 'tocco-ui'
+
+import {
+  StyledContainer,
+  StyledLogWrapper,
+  StyledHeader,
+  StyledLogFileStatedValue,
+  StyledFileCountStatedValue,
+  StyledButtonItem,
+  StyledTextAreaItem,
+  StyledTextarea
+} from './StyledComponents'
+
+const Log = ({data, fetchData, fileContent, fetchFileContent, intl}) => {
+  const [lines, setLines] = useState(500)
+  const [currentFile, setCurrentFile] = useState({key: 'nice.log', display: 'nice.log'})
+
+  const conditionalString = currentFile.key + '?lines=' + lines
+
+  useEffect(() => {
+    fetchData()
+  }, [fetchData])
+
+  // load file content on mount
+  useEffect(() => {
+    fetchFileContent(conditionalString)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [fetchFileContent])
+
+  const msg = id => intl.formatMessage({id})
+
+  const options = data ? data.map(fileNames => ({key: fileNames, display: fileNames})) : null
+  return (
+    <LoadMask required={[data, fileContent]}>
+      <StyledContainer>
+        <StyledLogWrapper>
+          <StyledHeader>
+            <StyledLogFileStatedValue>
+              <StatedValue label={msg('client.log.logfile')}>
+                <EditableValue
+                  type="single-select"
+                  value={currentFile}
+                  options={options ? {options: options} : null}
+                  events={{onChange: setCurrentFile}}
+                />
+              </StatedValue>
+            </StyledLogFileStatedValue>
+            <StyledFileCountStatedValue>
+              <StatedValue label={msg('client.log.filecount')}>
+                <EditableValue type="integer" value={lines} events={{onChange: setLines}} />
+              </StatedValue>
+            </StyledFileCountStatedValue>
+            <StyledFileCountStatedValue>
+              <StatedValue label={msg('client.log.hostname')}>
+                <EditableValue value={fileContent ? fileContent.hostName : null} type="string" readOnly={true} />
+              </StatedValue>
+            </StyledFileCountStatedValue>
+            <StyledButtonItem>
+              <Button ink="primary" look="raised" onClick={() => fetchFileContent(conditionalString)}>
+                <FormattedMessage id="client.log.reload" />
+              </Button>
+            </StyledButtonItem>
+          </StyledHeader>
+          <StyledTextAreaItem>
+            <StyledTextarea value={fileContent ? fileContent.fileContent : null} readOnly={true} />
+          </StyledTextAreaItem>
+        </StyledLogWrapper>
+      </StyledContainer>
+    </LoadMask>
+  )
+}
+
+Log.propTypes = {
+  fetchData: PropTypes.func,
+  fetchFileContent: PropTypes.func,
+  conditionalString: PropTypes.string,
+  data: PropTypes.array,
+  fileContent: PropTypes.shape({
+    fileContent: PropTypes.string,
+    hostName: PropTypes.string
+  }),
+  fileNames: PropTypes.array,
+  options: PropTypes.shape({
```

Comment:
todo: `options`, `conditionalString` and `fileNames` are no props

They are used values inside the component but no props. In the `propTypes` only component props have to be specified.